src/globus_cli/commands/streams/tunnel/show.py

```diff
@@ -0,0 +1,42 @@
+import uuid
+
+import click
+
+from globus_cli.commands.streams import streams_display_flatten_response
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import command
+from globus_cli.termio import Field, display
+
+STANDARD_FIELDS = [
+    Field("ID", "id"),
+    Field("Label", "attributes_label"),
+    Field("State", "attributes_state"),
+    Field("Status", "attributes_status"),
+    Field("Listener Stream AP", "relationships_listener_id"),
+    Field("Listener IP", "attributes_listener_ip_address"),
+    Field("Listener Port", "attributes_listener_port"),
+    Field("Initiator Stream AP", "relationships_initiator_id"),
+    Field("Initiator IP", "attributes_initiator_ip_address"),
+    Field("Initiator Port", "attributes_initiator_port"),
+]
```

Comment:
Could you pull these fields into a common module and re-use them across crud operations?

This will (intentionally) force the fieldset a users sees for create to mirror that of update, show etc.